homeassistant/components/coolmaster/__init__.py

```diff
@@ -1 +1,22 @@
-"""The coolmaster component."""
+"""The Coolmaster integration."""
+
+
+async def async_setup(hass, config):
+    """Set up Coolmaster components."""
+    return True
+
+
+async def async_setup_entry(hass, entry):
+    """Set up Coolmaster from a config entry."""
+    hass.async_add_job(hass.config_entries.async_forward_entry_setup(entry, "climate"))
+
+    return True
+
+
+async def async_unload_entry(hass, entry):
+    """Unload a Coolmaster config entry."""
+    await hass.async_add_job(
```

Comment:
We don't need to create a task here. Just await the unloading directly. Also return the awaited return value since it's a boolean indicating success.